src/core/SkCachedData.cpp

```diff
@@ -12,7 +12,8 @@
 #include "src/core/SkRecordReplay.h"
 
 SkCachedData::SkCachedData(void* data, size_t size)
-    : fData(data)
+    : fMutex("SkCachedData")
```

Comment:
This should be "SkCachedData.fMutex" to match the convention we're using.

Reply:
I stole the naming convention from `SkResourceCache`:

```cpp
static SkMutex& mutex = *(new SkMutex("SkResourceCache"));
```

I can fix that while I'm at it :shrug: